examples/README.rst

```diff
@@ -5,3 +5,12 @@
 Examples
 ========
 These examples demonstrate the behavior and usage of PyDPF - Composites.
+Some additional modules are required to run the examples which can be
+installed with the following command:
+
+.. code::
+
+    pip install ansys-dpf-composites[examples]
+
+It is recommended to run the examples in a Jupyter notebook or an IPython console.
+Using a standard Python shell can lead to formatting issues (e.g. IndentationError).
```

Comment:
```suggestion
Copy / pasting to a standard Python interactive shell can lead to formatting issues (e.g. IndentationError).
```
I think it makes sense to be specific about what can go wrong.

Also, this problem is fixed in Python 3.13 - do we want to mention that somehow? Or just add a comment to ourselves to remove this section once only 3.13+ is supported?